### ISSUE TITLE
Original language word lost when one word is split from a group of words

### ISSUE THREAD

**Issue by elsylambert** [unfoldingWord/gateway-edit#643]
When splitting original words, one of the original words gets lost.

<img width="426" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBL05NQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--2d48ff271fb15a7bc1e9fbb89473ffa2e8840016/Bug_Alignment_Splitting_Orignal_Words.gif" alt="Bug_Alignment_Splitting_Orignal_Words.gif" />

**Comment by elsylambert** [comment]
@PhotoNomad0 here is the issue for Aligner bug where a word is lost when split from a group of words.

**Comment by PhotoNomad0** [comment]
@elsylambert @danielklapp This should be fixed in v2.4.0 build e1d5248

**Comment by PhotoNomad0** [comment]
I cannot open this picture. Sent from my iPhoneOn May 16, 2024, at 8:51 AM, elsylambert ***@***.***> wrote:﻿
@PhotoNomad0 Not seeing loss of Hebrew word when splitting from a group of words, but after making changes, click on Accept and saving it on the card, seeing this error:

**Comment by PhotoNomad0** [comment]
@elsylambert I can't open your private Zenhub image and videos - tried following your steps, but am not seeing a problem.  Using QA server, tit 1:1 en_ust unfoldingWord.

**Comment by elsylambert** [comment]
(https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBd0h3QkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--189fc82531620eb580394d5db6badb142b7f9631/Screen%20Recording%202024-05-16%20at%208.53.50%E2%80%AFAM.mov)

**Comment by elsylambert** [comment]
[Private Zenhub Video](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeEx3QkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--d872295ba0b739d8dc8a54b705e906d43951e8f8/Screen%20Recording%202024-05-16%20at%208.53.50%E2%80%AFAM.mov)

**Comment by elsylambert** [comment]
@PhotoNomad0 Please check now if the screenshots and video is visible for you?^^

**Comment by danielklapp** [comment]
@PhotoNomad0, @elsylambert when I reproduce the steps here's my findings 
v2.4.0 build 5c6dfd8 QA
Don't see the loss of original words when splitting from a group of words, but getting a 409 (Conflict) error after clicking on Accept and saving it to the card. A weird thing happens when trying to merge my work. I can check the Mergeable Resources but can't leave a comment in the description box. Looking at the console it's giving a RangeError: Maximum call stack size exceeded.

<img width="1432" alt="Screenshot 2024-05-20 at 12 10 55 PM" src="https://github.com/unfoldingWord/gateway-edit/assets/82067076/2739d284-2015-45f5-9bad-3c550abf1d7e">
Here's my console history if it helps 

[deploy-preview-648--gateway-edit.netlify.app-1716221161445.log](https://github.com/unfoldingWord/gateway-edit/files/15379985/deploy-preview-648--gateway-edit.netlify.app-1716221161445.log)

**Comment by elsylambert** [comment]
Retested in v2.4.0 build 4b01301 QA this morning with a fresh branch and not seeing Network error. @PhotoNomad0

**Comment by elsylambert** [comment]
@danielklapp Can you retest with a fresh user branch? if the issue still exist, please write a new separate issue for that.

**Comment by danielklapp** [comment]
@elsylambert Retested with a fresh branch and issue still persist, will write up a separate issue for it.

**Comment by PhotoNomad0** [comment]
> @PhotoNomad0, @elsylambert when I reproduce the steps here's my findings v2.4.0 build [5c6dfd8](https://github.com/unfoldingWord/gateway-edit/commit/5c6dfd83ac91338217a8b182a091e77335342438) QA Don't see the loss of original words when splitting from a group of words, but getting a 409 (Conflict) error after clicking on Accept and saving it to the card. A weird thing happens when trying to merge my work. I can check the Mergeable Resources but can't leave a comment in the description box. Looking at the console it's giving a RangeError: Maximum call stack size exceeded.
> <img alt="Screenshot 2024-05-20 at 12 10 55 PM" width="1432" src="https://private-user-images.githubusercontent.com/82067076/332145467-2739d284-2015-45f5-9bad-3c550abf1d7e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTYzMjUwNDIsIm5iZiI6MTcxNjMyNDc0MiwicGF0aCI6Ii84MjA2NzA3Ni8zMzIxNDU0NjctMjczOWQyODQtMjAxNS00NWY1LTliYWQtM2M1NTBhYmYxZDdlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTIxVDIwNTIyMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTlhZjBkYTZjZWMyNDg5OWUyZTk4ZWU0MWQ2ZDVlNmJjZTQzNTBhY2E0ZmMyNTNkZWZhNWMxOTAyZWMzYTNjYWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.RP2Jg10h6iUDUD8Df2DS6D0uYMiVG_aaROuAB-GvOPE"> Here's my console history if it helps
> 
> [deploy-preview-648--gateway-edit.netlify.app-1716221161445.log](https://github.com/unfoldingWord/gateway-edit/files/15379985/deploy-preview-648--gateway-edit.netlify.app-1716221161445.log)

@danielklapp Yes, it looks like there is an endless recursive loop going on there causing a memory crash. @kintsoogi

**Comment by danielklapp** [comment]
Tested with v2.4.0 build 542041b QA, looks good to me. The issue I wrote up separate is issue #650.
